users/views.py

```diff
@@ -43,3 +43,21 @@ def post(self, *args, **kwargs):
 
         return render(self.request, self.template_name,
                       {'form':form}, status=400)
+
+
+class UpdateProfileView(LoginRequiredMixin, TemplateView):
```

Comment:
this doesn't make sense since I want the updating of the user profile to NOT refresh